conf/nebula-graphd.conf.default

```diff
@@ -7,7 +7,7 @@
 --enable_optimizer=true
 
 ########## logging ##########
-# The directory to host logging files, which must already exists
```

Comment:
But after this PR, it's optional.

Reply:
after this pr, if start the service with a script, the script will create a log directory, if don’t start the service with a script, the service creates a directory.